### DISCUSSION TITLE
Nvim Configs

### DISCUSSION THREAD

**Discussion by jpx40** [prasanthrangan/hyprdots#245] [Ideas]
Hey, could i add configs for Nvim to the repo?

**Comment by prasanthrangan** [comment]
Hey, Thank you so much! is it available for each theme or is it generated based on wallpaper?
But yes please feel free to commit your nvim configs to his repo... and i'll merge it!

**Reply by jpx40** [reply]
I have not written any configs yet but it was my goal do it someday. I will consider this repo then. But i am new to neovim. So it's mostly for learning purposes.

**Reply by prasanthrangan** [reply]
sure... sounds good, have fun!

**Reply by jpx40** [reply]
Sorry, i have my personal Config now, but i feel like the Config is to personal. When someone wants to use nvim he create his own personal Config.

**Reply by moguay** [reply]
A very good nvim conf : https://github.com/Matt-FTW/dotfiles/tree/main/.config/nvim
very well maintained and continuously improving

**Reply by jpx40** [reply]
i would recommend lazyvim, we have to customize it a lottle bit but it mostly ready

**Reply by prasanthrangan** [reply]
great, I'd like to include neovim/lazyvim as an option to vscode, but we should have color configs for each theme to sync

**Comment by jpx40** [comment]
Than let use lazyvim

-------- Original-Nachricht --------
Am 19. März 2024, 17:13, schrieb Tittu :

**Comment by jpx40** [comment]
created pull request https://github.com/prasanthrangan/hyprdots/pull/1122
